lib/manager/gradle/build-gradle.ts

```diff
@@ -387,3 +387,36 @@ export function updateGradleVersion(
   }
   return buildGradleContent;
 }
+
+const additionalKeywords = ['mavenBom'].join('|');
+
+const additionalKeywordsRegex = regEx(
+  `[\\s\\n{](${additionalKeywords})\\s*['"](?<depName>[a-zA-Z][-_a-zA-Z0-9.]*?:[a-zA-Z][-_a-zA-Z0-9.]*?):(?<currentValue>[a-zA-Z0-9][-.a-zA-Z0-9]*?)['"]`
+);
+
+export function extractAdditionalDeps(
+  content: string,
+  deps: BuildDependency[]
+): void {
+  const registryUrls = deps[0]?.registryUrls;
+  for (const line of content.split(/\s*\n+/)) {
+    const match = additionalKeywordsRegex.exec(line);
+    if (match) {
+      const { depName, currentValue } = match.groups;
+      const [name, depGroup] = depName.split(':');
+      if (
+        !deps.some(
+          (dep) => dep.depName === depName && dep.currentValue === currentValue
```

Comment:
If this dep is left out because it was detected previously, does that mean it also won't be updated later on?

Reply:
Do you mean because of custom update function? Need to double-check this case